WordPress/Classes/Stores/NoticeStore.swift

```diff
@@ -54,7 +54,7 @@ struct Notice {
 
     public enum Style {
         case normal
-        case quickStart
+        case quickStart(NSAttributedString?)
```

Comment:
I _think_ I'd prefer we just add an optional `formattedMessage` property to `Notice`, instead of having this on the Style 🤔 What do you think?

Reply:
Actually, perhaps at the same time we could rename it `attributedMessage`?

Reply:
I wasn't sure. The constructor for the `Notice` gets a bit heavy. And then later we'll be creating notices with two action buttons, and I figured—maybe if I shove those somewhere else it's less ugly 😉

But it could be wrong